mongodb/intro/date-aggregation/README.md

```diff
@@ -0,0 +1,10 @@
+name: Aggregation With Date
+
+description: Use date expression operators to extract date parts from the ObjectId into an ISODate
+
+insights:
+  - aggregation-with-date-expression-operators
+  - aggregation-with-date-expression-operators-ii
+  - aggregation-with-date-expression-operators-iii
+  - aggregation-with-date-expression-operators-iv
+  - aggregation-with-date-expression-operators-v
```

Comment:
this readme should contain the aspects

@Stefan-Stojanovic you need to add this workout to the course's README, i.e. in `mongodb/intro/README.md`